python/dgl/transform.py

```diff
@@ -1255,56 +1255,139 @@ def out_subgraph(g, nodes):
         ret.edges[etype].data[EID] = induced_edges[i].tousertensor()
     return ret
 
-def to_simple(g, return_counts='count', writeback_mapping=None):
-    """Convert a heterogeneous multigraph to a heterogeneous simple graph, coalescing
-    duplicate edges into one.
+def to_simple(g, return_counts='count', writeback_mapping=True, share_ndata=True):
+    r"""Convert a heterogeneous multigraph to a heterogeneous
+    simple graph, coalescing duplicate edges into one.
 
```

Comment:
Let's start with the case of graphs with a single edge type to make the learning curve more smooth.

Reply:
@classicsong , any comments?